### PR TITLE
#154066073 Create search page interface

### PR DESCRIPTION
**Pull request by andela-hms**
#### What does this PR do?
This PR creates the search page interface
#### Description of Task to be completed?
Create incidents search page Interface
#### How should this be manually tested?
- Run the project on your local machine
- On the dashboard, click on the search input field
- You should be redirected to the search page
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#154066073](https://www.pivotaltracker.com/n/projects/2117172/stories/154066073)
#### Screenshots (if appropriate)
screen shot 2018-01-16 at 2 30 46 pm

### CODE REVIEW

File: src/pages/Search/Search.Component.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+class SearchComponent extends Component {
+    constructor(props){
+        super(props);
+      }
+    
+    componentDidMount(){
+        this.nameInput.focus();
+      }
+
```
Comment (jimmykimani) [inline src/pages/Search/Search.Component.jsx]: Lets add a JSdoc to document the method and the necessary params

File: src/pages/Search/Search.Component.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                <TextField
+                    ref={(input) => { this.nameInput = input; }}
+                    floatingLabelText="Search for an incident"
+                    fullWidth
+                    rows={2}
+                    className="input-style"
+                />
+                <i className="fa fa-times-circle" title="Click to exit search" onClick={this.handleExit} />
+            </div>
+
```
Comment (jimmykimani) [inline src/pages/Search/Search.Component.jsx]: Line 33 is unnecessary

File: src/pages/Search/Search.Component.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+class SearchComponent extends Component {
+    constructor(props){
+        super(props);
+      }
+    
+    componentDidMount(){
+        this.nameInput.focus();
+      }
+
+    handleExit = (event) => {
```
Comment (jimmykimani) [inline src/pages/Search/Search.Component.jsx]: Lets add a JSdoc to document the method and the necessary params

File: src/pages/Search/Search.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+.fa-times-circle:hover {
+    color: map_get($red, 300);
+}
+
+.fa-times-circle {
+    float: right;
+    position: absolute;
+    margin-top: 2.5vh;
+    color: map_get($grey, 200);
+    font-size: 1.7rem !important;
```
Comment (jimmykimani) [inline src/pages/Search/Search.scss]: Add the cursor property to pointer

File: src/pages/Dashboard/Dashboard.Component.jsx [hunk 5]
```diff
@@ -26,14 +26,13 @@ class Dashboard extends Component {
   componentDidMount() {
     this.props.loadIncidents();
   }
-
   render() {
-
     const { incidents } = this.props;
-    
+
```
Comment (Mercy-Muchai) [inline src/pages/Dashboard/Dashboard.Component.jsx]: Trailing white spaces

**Pull request merged**